importApp/src/main/java/com/commercetools/dataimport/joyrideavailability/AvailabilityJobConfiguration.java

```diff
@@ -0,0 +1,246 @@
+                                  final ItemReader<ChannelDraftDsl> channelsDraftReader,
+                                  final ItemWriter<ChannelDraftDsl> channelsDraftItemWriter) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("channelImportStep");
+        return stepBuilder
+                .<ChannelDraftDsl, ChannelDraftDsl>chunk(1)
+                .reader(channelsDraftReader)
+                .writer(channelsDraftWriter(sphereClient))
+                .build();
+    }
+
+    @Bean
+    public Step createInventoryEntryStep(final BlockingSphereClient sphereClient,
+                                         final ItemProcessor<Product, List<InventoryEntryDraft>> inventoryEntryProcessor,
+                                         final ItemWriter<List<InventoryEntryDraft>> inventoryEntryWriter) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("createInventoryEntryStep");
+        return stepBuilder
+                .<Product, List<InventoryEntryDraft>>chunk(productsImportStepChunkSize)
+                .reader(ItemReaderFactory.sortedByIdQueryReader(sphereClient, ProductQuery.of()))
+                .processor(inventoryEntryProcessor(sphereClient))
+                .writer(inventoryEntryWriter)
+                .build();
+    }
+
+    @Bean
+    public Step setPricesStep(final BlockingSphereClient sphereClient,
+                              final ItemWriter<ProductUpdateCommand> setPriceWriter) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("createInventoryEntryStep");
+        return stepBuilder
+                .<Product, ProductUpdateCommand>chunk(productsImportStepChunkSize)
+                .reader(ItemReaderFactory.sortedByIdQueryReader(sphereClient, ProductQuery.of()))
+                .processor(setPriceProcessor(sphereClient))
+                .writer(setPriceWriter)
+                .build();
+    }
+
+    @Bean
+    @StepScope
+    private ItemReader<TypeDraft> customTypeReader(@Value("#{jobParameters['typesResource']}") final Resource typesJsonResource) throws IOException {
+        logger.info("URL_Types: " + typesJsonResource);
+        final ObjectReader reader = SphereJsonUtils.newObjectMapper().readerFor(new TypeReference<List<TypeDraft>>() { });
+        final InputStream inputStream = typesJsonResource.getInputStream();
+        final List<TypeDraft> list = reader.readValue(inputStream);
+        return new ListItemReader<>(list);
+    }
+
+    @Bean
+    protected ItemWriter<TypeDraft> customTypeWriter(final BlockingSphereClient sphereClient) {
+        return items -> items.forEach(customType -> sphereClient.executeBlocking(TypeCreateCommand.of(customType)));
+    }
+
+    @Bean
+    @StepScope
+    private ItemReader<ChannelDraftDsl> channelsDraftReader(@Value("#{jobParameters['channelsResource']}") final Resource channelsJsonResource) throws IOException {
+        logger.info("URL_Channels: " + channelsJsonResource);
```

Comment:
It is not necessary to log the job parameters explicitly, spring does it, it looks like this:

```
11:50:55.645 [main] INFO  o.s.b.c.l.support.SimpleJobLauncher - Job: [SimpleJob: [name=importJoyrideChannelsJob]] launched with the following parameters: [{commercetools.projectKey=joyride-playground-1, commercetools.clientId=EpIQOiYfutXCLIlDLqFMmIYn, commercetools.clientSecret=*Iremoveditfromtheoutput*, commercetools.authUrl=https://auth.sphere.io, commercetools.apiUrl=https://api.sphere.io, name=importJoyrideChannelsJob, channelsResource=file:///Users/mschleichardt/dev/commercetools-sunrise-data/joyride/channels.json, typesResource=file:///Users/mschleichardt/dev/commercetools-sunrise-data/joyride/types.json}]
```